netbox/dcim/urls.py

```diff
@@ -177,6 +177,8 @@
     url(r'^interface-connections/(?P<pk>\d+)/delete/$', views.interfaceconnection_delete, name='interfaceconnection_delete'),
     url(r'^interfaces/(?P<pk>\d+)/edit/$', views.InterfaceEditView.as_view(), name='interface_edit'),
     url(r'^interfaces/(?P<pk>\d+)/delete/$', views.InterfaceDeleteView.as_view(), name='interface_delete'),
+    url(r'^interfaces/$', views.InterfaceListView.as_view(), name='interface_list'),
```

Comment:
There's no need for a view which exposes device components outside the context of a device.